app/src/lib/stores/app-store.ts

```diff
@@ -5105,16 +5115,21 @@ export class AppStore extends TypedBaseStore<IAppState> {
     repository: Repository,
     branch: Branch
   ): Promise<void> {
-    const gitHubRepository = repository.gitHubRepository
+    const { gitHubRepository } = repository
+
     if (!gitHubRepository) {
       return
     }
 
-    const urlEncodedBranchName = escape(branch.nameWithoutRemote)
-    const baseURL = `${
-      gitHubRepository.htmlURL
-    }/pull/new/${urlEncodedBranchName}`
+    const { parent: parentGitHubRepository } = gitHubRepository
 
+    const urlEncodedBranchName = escape(branch.nameWithoutRemote)
+    const baseURL =
+      repository.isGitHubFork && parentGitHubRepository
```

Comment:
I think this should be:

```suggestion
      repository.isGitHubFork() && parentGitHubRepository
```

since `isGitHubFork` is a method.